app/sections/CountDown/ButtonItems.tsx

```diff
@@ -0,0 +1,60 @@
+import type {
+  HydrogenComponentProps,
+  HydrogenComponentSchema,
+} from '@weaverse/hydrogen';
+import { forwardRef, CSSProperties } from 'react';
+
+interface ButtonItemsProps extends HydrogenComponentProps {
+  gap: number;
+}
+
+let ButtonItems = forwardRef<HTMLDivElement, ButtonItemsProps>((props, ref) => {
+  let { gap, children, ...rest } = props;
+  let spacingStyle: CSSProperties = {
+    gap: `${gap}px`,
+  } as CSSProperties;
+  return (
+    <div ref={ref} {...rest} className='flex mt-3' style={spacingStyle}>
+      {children}
+    </div>
+  );
+});
+
+export default ButtonItems;
+
+export let schema: HydrogenComponentSchema = {
+  type: 'Button-CountDown-items',
```

Comment:
Use lowercase text for all items `type`. `Button-CountDown-items` --> `countdown-buttons`